app/controllers/transcriptions_controller.rb

```diff
@@ -13,21 +19,24 @@ def index
   def show
     @transcription = Transcription.find(params[:id])
     authorize @transcription
+
+    if TranscriptionPolicy.new(current_user, @transcription).has_update_rights?
+      @transcription.lock(current_user)
+    end
+
+    response.set_header('Last-Modified', @transcription.updated_at.rfc2822)
```

Comment:
I don't think you have to format this specifically using the `rfc2822`, i think that's the default. Worth testing to see if the string format on serialize out matches the format and if so remove the extra method.

Reply:
Ive updated the code to mirror how things are done in Rails as much as possible, am now transforming the date via the `.httpdate` method to imitate the act of retrieving the date value from the "Last-Modified" response header. When removing the transformation, it fails, so I'll be keeping it on.